src/helpers/parseNumbers.ts

```diff
@@ -10,17 +17,37 @@ BigNumber.NE = -20;
 // Converts a decimal number to the native representation (a large integer)
 export const decimalToNative = (value: BigNumber | number | string) => {
   const bigIntValue = new BigNumber(value);
-  const multiplier = new BigNumber(10).pow(TOKEN_DECIMALS);
+  const multiplier = new BigNumber(10).pow(AmplitudeDecimals);
```

Comment:
Okay with the lowercase buuut, is it only amplitude? I think we have 12 decimals in all our chains.

Reply:
True. We could rename it to `ChainDecimals` maybe and neglect the fact that this could be different for each chain until we actually encounter the case where we need to use different decimals (which is unlikely to happen anytime soon).